### ISSUE TITLE
UnicodeDecodeError

### ISSUE THREAD

**Issue by hoopty** [clinton-hall/nzbToMedia#941]
I've seen this before, getting the following error on multiple files:
UnicodeDecodeError: 'utf8' codec can't decode byte 0xa9 in position 0: invalid start byte

I've tried with convert=0 and convert=1 (setting the log below is from):

```
Wed Jan 20 22:08:10 2016        INFO    Spectre.2015.1080p.BluRay.x264.DTS.cp(tt2379713) returned from history back to download queue
Wed Jan 20 22:08:10 2016        INFO    Queueing Spectre.2015.1080p.BluRay.x264.DTS.cp(tt2379713) for post-processing
Wed Jan 20 22:08:10 2016        INFO    Bridge.of.Spies.2015.1080p.BluRay.x264.DTS.cp(tt3682448) returned from history back to download queue
Wed Jan 20 22:08:10 2016        INFO    Queueing Bridge.of.Spies.2015.1080p.BluRay.x264.DTS.cp(tt3682448) for post-processing
Wed Jan 20 22:08:10 2016        INFO    Executing post-process-script nzbToMedia/nzbToMedia.py for Bridge.of.Spies.2015.1080p.BluRay.x264.DTS.cp(tt3682448)
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: Checking database structure...
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: Checking if git needs an update
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: No update needed
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: nzbToMedia Version:3e4861e87c543a4d2debd8997feb16bda3ef5339 Branch:master (FreeBSD
10.2-RELEASE-p9)
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: #########################################################
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: ## ..::[nzbToMedia.py]::.. ##
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: #########################################################
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: Script triggered from NZBGet Version 16.4.
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: [22:08:11] [INFO]::MAIN: Auto-detected SECTION:CouchPotato
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: /usr/local/nzbget/scripts/nzbToMedia/core/nzbToMediaUtil.py:366: UnicodeWarning: Unicode equal comparison fa
iled to convert both arguments to Unicode - interpreting them as being unequal
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:   if ((Name[Idx] == '\xC2') | (Name[Idx] == '\xC3')) & ((Name[Idx+1] >= '\xA0') & (Name[Idx+1] <= '\xFF')):
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: Traceback (most recent call last):
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 796, in <module>
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:     exit(main(sys.argv))
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 698, in main
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:     failureLink=failureLink)
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 585, in process
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:     inputName, inputDirectory = convert_to_ascii(inputName, inputDirectory)
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/core/nzbToMediaUtil.py", line 420, in convert_to_ascii
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:     encoded, filename2 = CharReplace(filename)
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/core/nzbToMediaUtil.py", line 370, in CharReplace
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:     elif (Name[Idx] >= '\x80') & (Name[Idx] <= '\xA5'):
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:   File "/usr/local/lib/python2.7/encodings/utf_8.py", line 16, in decode
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia:     return codecs.utf_8_decode(input, errors, True)
Wed Jan 20 22:08:11 2016        INFO    nzbToMedia: UnicodeDecodeError: 'utf8' codec can't decode byte 0xa9 in position 0: invalid start byte
Wed Jan 20 22:08:11 2016        ERROR   Post-process-script nzbToMedia/nzbToMedia.py for Bridge.of.Spies.2015.1080p.BluRay.x264.DTS.cp(tt3682448) failed (termin
ated with unknown status)
Wed Jan 20 22:08:11 2016        INFO    Collection Bridge.of.Spies.2015.1080p.BluRay.x264.DTS.cp(tt3682448) added to history
Wed Jan 20 22:08:11 2016        INFO    Executing post-process-script nzbToMedia/nzbToMedia.py for Spectre.2015.1080p.BluRay.x264.DTS.cp(tt2379713)
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: Checking database structure...
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: Checking if git needs an update
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: No update needed
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: nzbToMedia Version:3e4861e87c543a4d2debd8997feb16bda3ef5339 Branch:master (FreeBSD
10.2-RELEASE-p9)
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: #########################################################
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: ## ..::[nzbToMedia.py]::.. ##
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: #########################################################
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: Script triggered from NZBGet Version 16.4.
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: [22:08:12] [INFO]::MAIN: Auto-detected SECTION:CouchPotato
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: /usr/local/nzbget/scripts/nzbToMedia/core/nzbToMediaUtil.py:366: UnicodeWarning: Unicode equal comparison fa
iled to convert both arguments to Unicode - interpreting them as being unequal
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:   if ((Name[Idx] == '\xC2') | (Name[Idx] == '\xC3')) & ((Name[Idx+1] >= '\xA0') & (Name[Idx+1] <= '\xFF')):
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: Traceback (most recent call last):
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 796, in <module>
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:     exit(main(sys.argv))
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 698, in main
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:     failureLink=failureLink)
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/nzbToMedia.py", line 585, in process
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:     inputName, inputDirectory = convert_to_ascii(inputName, inputDirectory)
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/core/nzbToMediaUtil.py", line 420, in convert_to_ascii
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:     encoded, filename2 = CharReplace(filename)
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:   File "/usr/local/nzbget/scripts/nzbToMedia/core/nzbToMediaUtil.py", line 370, in CharReplace
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:     elif (Name[Idx] >= '\x80') & (Name[Idx] <= '\xA5'):
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:   File "/usr/local/lib/python2.7/encodings/utf_8.py", line 16, in decode
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia:     return codecs.utf_8_decode(input, errors, True)
Wed Jan 20 22:08:12 2016        INFO    nzbToMedia: UnicodeDecodeError: 'utf8' codec can't decode byte 0xa9 in position 0: invalid start byte
Wed Jan 20 22:08:12 2016        ERROR   Post-process-script nzbToMedia/nzbToMedia.py for Spectre.2015.1080p.BluRay.x264.DTS.cp(tt2379713) failed (terminated wit
h unknown status)
Wed Jan 20 22:08:13 2016        INFO    Collection Spectre.2015.1080p.BluRay.x264.DTS.cp(tt2379713) added to history
```

**Comment by clinton-hall** [comment]
In autoProcessMedia.cfg please set [General] log_debug = 1, log_env = 1 and post the logs from the next run.

From the 2 examples above, what are the file names?

**Comment by mikerohan** [comment]
Similar problem here...

**Comment by clinton-hall** [comment]
Applied a possible fix in #954 that may help out this issue too.
